### DISCUSSION TITLE
🆕 Software and functions you would like to see implemented

### DISCUSSION THREAD

**Discussion by DesobedienteTecnologico** [DesobedienteTecnologico/dtails#31] [Ideas]
[Create a Issue  here](https://github.com/DesobedienteTecnologico/dtails/issues)
---
Add the following information:

**Title:** Add [Software]
**Description:**
**Software name:**
**Link to download:**
**Verification (Signature, GPG ...):**
**Manual configuration needed?** NO / YES (elaborate)

**Comment by ivanwmco** [comment]
Could you add the _**latest versions**_ of the following applications to the list:

- Electrum
- [Electrum-Satochip](https://satochip.io/setup-use-satochip-with-electrum-bitcoin/) (for as long as Electrum doesn't support Satochip yet; pull requests: to be added in 4.6.0 
milestone)
- KeePassXC

**Reply by DesobedienteTecnologico** [reply]
Could you please create an issue for each proposal individually?
